Neuro/Utils/Methods.cs

```diff
@@ -48,4 +49,63 @@ public static string GetLocationFromPosition(Vector2 position)
 
         return closestLocation;
     }
+
+    public static (float min, float max) TaskTypeToTimeToCompleteTask(TaskTypes taskTypes) => MiniGameTimes.ContainsKey(taskTypes) ? MiniGameTimes[taskTypes] : new (2f,4f); 
+
+    
+    private static Dictionary<TaskTypes, (float min, float max)> MiniGameTimes = new Dictionary<TaskTypes, (float min, float max)>
+    {
+        {TaskTypes.AlignEngineOutput, new (10f,15f)},
+        {TaskTypes.AlignTelescope, new (2f, 4f)},
+        {TaskTypes.AssembleArtifact, new (2f, 4f)},
+        {TaskTypes.BuyBeverage, new (2f, 4f)},
+        {TaskTypes.CalibrateDistributor, new (2f, 4f)},
+        {TaskTypes.ChartCourse, new (2f, 4f)},
+        {TaskTypes.CleanO2Filter, new (2f, 4f)},
+        {TaskTypes.CleanToilet, new (2f, 4f)},
+        {TaskTypes.VentCleaning, new (2f, 4f)},
+        {TaskTypes.ClearAsteroids, new (10f, 15f)},
+        {TaskTypes.Decontaminate, new (2f, 4f)},
+        {TaskTypes.DevelopPhotos, new (10f, 15f)},
+        {TaskTypes.DivertPower, new (2f, 4f)},
+        {TaskTypes.DressMannequin, new (2f, 4f)},
+        {TaskTypes.EmptyChute, new (10f, 15f)},
+        {TaskTypes.EmptyGarbage, new (10f, 15f)},
+        {TaskTypes.EnterIdCode, new (5f, 8f)},
+        {TaskTypes.FillCanisters, new (2f, 4f)},
+        {TaskTypes.FixShower, new (2f, 4f)},
+        {TaskTypes.FixWiring, new (5f, 8f)},
+        {TaskTypes.FuelEngines, new (10f, 15f)},
+        {TaskTypes.InsertKeys, new (5f, 8f)},
+        {TaskTypes.InspectSample, new (10f, 15f)},
+        {TaskTypes.MakeBurger, new (2f, 4f)},
+        {TaskTypes.MeasureWeather, new (2f, 4f)},
+        {TaskTypes.OpenWaterways, new (10f, 15f)},
+        {TaskTypes.PickUpTowels, new (2f, 4f)},
+        {TaskTypes.PolishRuby, new (2f, 4f)},
+        {TaskTypes.PrimeShields, new (2f, 4f)},
+        {TaskTypes.ProcessData, new (2f, 4f)},
+        {TaskTypes.PutAwayPistols, new (2f, 4f)},
+        {TaskTypes.PutAwayRifles, new (2f, 4f)},
```

Comment:
> Tasks that don't have a minigame will probably break something.
There are already breaks in the code on some minigames. This wasn't pushed to address them.

> This includes the Airship tasks for putting away pistols, rifles, and cleaning the towels. 
Tested they work fine there is a minigame after you pick them up when you turn them in.

Reply:
> There are already breaks in the code on some minigames. This wasn't pushed to address them.

Oh yeah the leaves minigame on Polus counts each leaf as one step. Also getting the next location for the align engine task is broken because it always returns the same location (upper engine i think)